src/email-templates/offer_disabled_notification.handlebars

```diff
@@ -0,0 +1,9 @@
+<p>Hello {{companyName}},</p>
+<br>
+<p>We took a look at your offer for {{offerTitle}} and decided to disable it.</p>
+<p>Some common causes for this are spam, abuse or a type of offer that does not align with our principles.
+    If you believe this was a mistake and want to discuss this decision, please get back at us at
+    <a href="mailto:ni@aefeup.pt">ni@aefeup.pt</a></p>
+<br>
```

Comment:
I think we should mention the offer content, to keep a record of what it was when it was disabled.

We can have a `<blockquote>` or something with the content.

This will make sure that even after the offer gets edited and enabled back, we still have the emails to look for examples of "bad" offers, while we don't have a log system to show every edit made (offer history)